File: R/tests/testthat/test_bart_ewmv_minustau.R
context("Test bart_ewmv_minustau")
library(hBayesDM)

test_that("Test bart_ewmv_minustau", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_ewmv_minustau(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_bart_ewmv_minustau.py
import pytest

from hbayesdm.models import bart_ewmv_minustau


def test_bart_ewmv_minustau():
    _ = bart_ewmv_minustau(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
